src/Service.Host/client/src/components/IntegerUpdater.js

```diff
@@ -35,11 +35,20 @@ export class IntegerUpdater extends Component {
         this.setState({ currentValue: value });
     }
 
-    handleOKClick() {
+    handleOkClick() {
         this.setState({ isOpen: false });
         this.props.onChange(this.state.currentValue);
     }
 
+    handleKeyPress(keyCode) {        
+        if (!this.saveDisabled() && keyCode === 13) {            
```

Comment:
Can we have these in a constants file somewhere please? That way the codes can be referred to by sensible names such as `ESC_KEY`, `RETURN_KEY`